addons/project/views/project_sharing_project_task_views.xml

```diff
@@ -212,36 +221,30 @@
                                     <field name="milestone_id"
                                         optional="hide"
                                         context="{'default_project_id': project_id}"
                                         attrs="{'invisible': [('allow_milestones', '=', False)], 'column_invisible': [('parent.allow_milestones', '=', False)]}"
                                     />
-                                    <field name="partner_id" optional="hide"/>
-                                    <field name="parent_id" optional="hide" groups="base.group_no_one"/>
-                                    <field name="user_ids" widget="many2many_avatar_user" optional="show" domain="[('share', '=', False), ('active', '=', True)]"/>
-                                    <field name="company_id" optional="hide" groups="base.group_multi_company" />
                                     <field name="company_id" invisible="1"/>
-                                    <field name="activity_ids" string="Next Activity" widget="list_activity" optional="hide"/>
-                                    <field name="date_deadline" attrs="{'invisible': [('is_closed', '=', True)]}" optional="show" />
+                                    <field name="partner_id" options="{'no_open': True, 'no_create': True, 'no_edit': True}" optional="hide" attrs="{'invisible': [('project_id', '=', False)]}"/>
+                                    <field name="portal_user_names" string="Assignees" optional="show"/>
+                                    <field name="date_deadline" attrs="{'invisible': [('state', 'in', ['1_done', '1_canceled'])]}" optional="hide"/>
                                     <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color'}" optional="hide"/>
-                                    <field name="rating_last_text" string="Rating" decoration-danger="rating_last_text == 'ko'"
-                                        decoration-warning="rating_last_text == 'ok'" decoration-success="rating_last_text == 'top'"
-                                        attrs="{'invisible': [('rating_last_text', '=', 'none')]}"
-                                        class="fw-bold" widget="badge" optional="hide"/>
-                                    <field name="stage_id" optional="show" invisible="1"/> 
-                                    <button class="oe_link float-end" string="View Task" name="action_open_task" type="object"/>
+                                    <field name="stage_id" optional="show"/>
+                                    <button name="action_open_task" type="object" title="View Task" string="View Task" class="btn btn-link float-end"
+                                            context="{'form_view_ref': 'project.project_sharing_project_task_view_form'}"
+                                            attrs="{'invisible': &quot;[('project_id', '!=', False), ('project_id', '!=', active_id)]&quot;}"/>
```

Comment:
The `&quot;` are not necessary here I believe

Reply:
mmh they are necessary :thinking:
it seems like this is because we are in a button tag not a field one